importers.js

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
Thoughts on putting this in src and as part of building copy it into lib? Same for wrappers.mjs.

Reply:
Good point - done!